src/models/methods/baseMethod.ts

```diff
@@ -0,0 +1,24 @@
+export interface BaseRequest {
+    id: number | string
+    command: string
+    api_version?: number
+}
+
+interface Warning {
+    id: number
+    message: string
+    details?: {[key: string]: string}
+}
+
+export interface BaseResponse {
+    id: number | string
+    status: string
+    type: string
+    result: any
+    warning?: "load"
+    warnings?: Warning[]
+    forwarded?: boolean
+    error?: string
+    request?: any
```

Comment:
Could this be type `BaseResponse`? IIRC, request just echos back the request sent to the server...

Reply:
Good point

Reply:
No, it would actually have to be `Response`